bazel/linux/uml.bzl

```diff
@@ -18,16 +18,18 @@ else
 fi
 
 # If debugging is enabled, throw the user in a shell.
-if [ -z "$INTERACTIVE" ]; then
-  OPTIONS=("con0=null,fd:1" "con1=null,fd:1" "${{OPTIONS[@]}}")
-else
+if [ "$SINGLE" ]; then
```

Comment:
`if [ -z "$SINGLE" -a -z "$INTERACTIVE" ]; then`...

and remove the elif below? also, forgot the `-z`?

Reply:
Rewrote this piece of code to make it more clear. How does it look now?